yahoo-japan/road-network.json

```diff
@@ -0,0 +1,151 @@
+{
+  "type": "FeatureCollection",
+  "name": "road-network",
+  "data_layers": [],
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [136.642, 35.096]
+      },
+      "properties": {
+        "place_name": "Japan",
+        "zoom": 8,
+        "description": "Major road networks and highway shields."
```

Comment:
Hmm I'm seeing this coordinates pointing to empty land

Reply:
Again because of a random at z8, but I just changed to a more accurate location if someone zooms in!